package.json

```diff
@@ -16,14 +16,14 @@
   },
   "dependencies": {
     "@babel/compat-data": "^7.19.1",
-    "@bcrs-shared-components/breadcrumb": "2.1.1",
+    "@bcrs-shared-components/breadcrumb": "2.1.2",
     "@bcrs-shared-components/confirm-dialog": "1.2.1",
-    "@bcrs-shared-components/corp-type-module": "1.0.8",
-    "@bcrs-shared-components/court-order-poa": "2.1.1",
-    "@bcrs-shared-components/enums": "1.0.23",
+    "@bcrs-shared-components/corp-type-module": "1.0.9",
+    "@bcrs-shared-components/court-order-poa": "2.1.2",
+    "@bcrs-shared-components/enums": "1.0.24",
     "@bcrs-shared-components/interfaces": "1.0.44",
-    "@bcrs-shared-components/staff-comments": "1.3.1",
-    "@bcrs-shared-components/staff-payment": "2.1.1",
+    "@bcrs-shared-components/staff-comments": "1.3.2",
+    "@bcrs-shared-components/staff-payment": "2.1.2",
```

Comment:
There are "so many" updates because they all depend on shared enums, which depends on shared corp-type-module. I verified that this is the only actual change.